src/style.css

```
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: inline-block;
+  height: 12px;
+  width: 12px;
+  margin: 2px 6px 2px 2px;
+  background-image: url("/images/star.svg");
+  background-size: cover;
+}
+
+.stars__star:nth-child(n) {
+  background-image: url("/images/star.svg");
+}
+
+.stars--0 .stars__star:nth-child(-n+0) {
+  background-image: url("/images/star-active.svg");
+}
+
+.stars--1 .stars__star:nth-child(-n+1) {
+  background-image: url("/images/star-active.svg");
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url("/images/star-active.svg");
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url("/images/star-active.svg");
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url("/images/star-active.svg");
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url("/images/star-active.svg");
+}
```

let's group these, because the styles applied are the same

```suggestion
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--2 .stars__star:nth-child(-n+3),
.stars--2 .stars__star:nth-child(-n+4),
.stars--2 .stars__star:nth-child(-n+5), {
  background-image: url("/images/star-active.svg");
}
```

---

Done.

---

I basically changed my old code per your suggestion without thinking much of what I'm writing. Need to pay more attention in the future.